From 2e17ababbfaea8efefd0d534a2c83b6ae9b0a410 Mon Sep 17 00:00:00 2001
From: Arthur Loiret <arthur.loiret@gmail.com>
Date: Sat, 13 Mar 2010 18:11:06 +0100
Subject: Package support nostrip flag at build

Make the build process support the nostrip flag in DEB_BUILD_OPTIONS.

Origin: vendor
Bug-Debian: http://bugs.debian.org/438097
Forwarded: not-needed
Last-Update: 2010-04-20
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index 5af111e..53afb0b 100644
--- a/Makefile
+++ b/Makefile
@@ -202,7 +202,7 @@ INSTALL=install
 ifndef CONFIG_WIN32
 install: $(PROGS) $(LIBTCC1) $(BCHECK_O) libtcc.a tcc.1 tcc-doc.html
 	mkdir -p "$(bindir)"
-	$(INSTALL) -s -m755 $(PROGS) "$(bindir)"
+	$(INSTALL) -m755 $(PROGS) "$(bindir)"
 	mkdir -p "$(mandir)/man1"
 	$(INSTALL) tcc.1 "$(mandir)/man1"
 	mkdir -p "$(tccdir)"
-- 
1.7.5.4

